src/hipscat/io/file_io/file_io.py

```diff
@@ -37,6 +37,22 @@ def make_directory(
     file_system.makedirs(file_pointer, exist_ok=exist_ok)
 
 
+def handle_pandas_storage_options(
```

Comment:
```suggestion
def unnest_headers_for_pandas(
```

Just one suggestion to clarify the intent for this method.